packages/server-renderer/src/renderToStream.ts

```diff
@@ -62,7 +66,7 @@ export function renderToStream(
   // provide the ssr context to the tree
   input.provide(ssrContextKey, context)
 
-  const stream = new Readable()
+  const stream = new PushSafeReadable()
```

Comment:
Not sure if this is the right fix anyway but shouldn't this be

```suggestion
  const stream = new Readable({
    read() {
      // the implementation
    }
  })
```

Reply:
I think it's okay